guides/common/modules/proc_creating-a-host.adoc

```diff
@@ -4,6 +4,9 @@
 Use this procedure to create a host in {ProjectName}.
 To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-host_{context}[].
 
+.Prerequisites
+* Review the provider-specific procedures for creating a host in the {ProvisioningDocURL}[Provisioning guide].
```

Comment:
This is not a prerequisite. If the provider-specific procedure exists in the Provisioning guide, the user does not need to follow this generic procedure.

Reply:
I agree with Avital. I'd suggest removing most of this procedure anyway because there are many requirements to be met before user can start creating hosts in order to provision them. I would only keep the link and delete the procedure steps personally.